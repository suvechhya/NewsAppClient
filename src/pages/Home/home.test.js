import { shallow } from 'enzyme';
import SearchBar from './Searchbar';

describe('SearchBar', () => {

    it('SearchBar should render correctly with props', () => {
        const clickFn = jest.fn();
        const component = shallow(<SearchBar title={'Test'} searchKeyword={clickFn} />);
        expect(component).toMatchSnapshot();
    });

    it('should have the text News atOnce', () => {
        const clickFn = jest.fn();
        const component = shallow(<SearchBar title={'Test'} searchKeyword={clickFn} />);
        expect(component.childAt(0).contains("Test")).toEqual(true);
    });
});
