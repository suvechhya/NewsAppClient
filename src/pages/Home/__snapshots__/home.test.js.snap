// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home SearchBar should render correctly with props 1`] = `ShallowWrapper {}`;

exports[`SearchBar SearchBar should render correctly with props 1`] = `ShallowWrapper {}`;
